packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -143,7 +143,7 @@ export default class Meeting extends StatelessSparkPlugin {
      * @private
      * @memberof Meeting
      */
-    this.roapSeq = 0;
+    this.roapSeq = -1;
```

Comment:
hmmmmmmm. why?

Reply:
The first one is always 0 and the code increments it . And we need to know if there was no offer sent for the first time so we had to change it to -1

Reply:
we should name the magic number to ROAP_SEQ_PRE and add the comment there for why it's -1

Reply:
done . I mis read the previous comment . I thought you mentioned about renaming it